5.5/apache/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM php:5.5-apache
+
+ENV	XHPROF_VERSION 0.9.4
+
+RUN apt-get update && apt-get install -y \
+        libfreetype6-dev \
+        libjpeg62-turbo-dev \
+        libmcrypt-dev \
+        libpng12-dev \
+    && docker-php-ext-install mysql mysqli pdo pdo_mysql mcrypt zip \
+    && docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/ \
+    && docker-php-ext-install -j$(nproc) gd exif \
+    && a2enmod rewrite \
+    && a2enmod proxy_http
+
+RUN	docker-php-source extract \
```

Comment:
Hi, 

I would remove the XHPROF stuff from the base image so it's kept as light as possible.  We've the XHPROF stuff in our other dev images so don't think we're using it so can probably drop it completely.

Many thanks.

Chris.